Building sormas with "with-app" profile fails

### Problem Description

When building sormas with the "with-app" profile using ```mvn install -Pwith-app```, the build fails with an error.

### Reproduction Steps

1. run ```mvn install -Pwith-app``` from sormas-base
2. the build fails with following error:

```[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[FATAL] Non-resolvable parent POM for de.symeda.sormas:sormas-app:1.97.0-SNAPSHOT: The following artifacts could not be resolved: de.symeda.sormas:sormas-base:pom:1.97.0-SNAPSHOT (absent): Could not find artifact de.symeda.sormas:sormas-base:pom:1.97.0-SNAPSHOT and 'parent.relativePath' points at wrong local POM @ line 3, column 10
 @ 
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project de.symeda.sormas:sormas-app:1.97.0-SNAPSHOT (/home/jenkins/sormas-project/SORMAS-Project/sormas-app/pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for de.symeda.sormas:sormas-app:1.97.0-SNAPSHOT: The following artifacts could not be resolved: de.symeda.sormas:sormas-base:pom:1.97.0-SNAPSHOT (absent): Could not find artifact de.symeda.sormas:sormas-base:pom:1.97.0-SNAPSHOT and 'parent.relativePath' points at wrong local POM @ line 3, column 10
```


### Expected Behavior

Build of project including the android app is successful.

### Affected Area(s)

Android app

### SORMAS Version

1.98.0-SNAPSHOT

### Android version/Browser

maven

### User Role(s)

No Response

### Server URL

_No response_

### Additional Information

_No response_

--- a/sormas-app/pom.xml
+++ b/sormas-app/pom.xml
@@ -3,7 +3,7 @@
 	<parent>
 		<artifactId>sormas-base</artifactId>
 		<groupId>de.symeda.sormas</groupId>
-		<version>1.97.0-SNAPSHOT</version>
+		<version>1.98.0-SNAPSHOT</version>
 		<relativePath>../sormas-base</relativePath>
 	</parent>
 	<modelVersion>4.0.0</modelVersion>

